client/state/data-layer/wpcom/sites/exports/media/test/index.js

```diff
@@ -0,0 +1,65 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { NOTICE_CREATE } from 'state/action-types';
+import { setMediaExportData } from 'state/site-settings/exporter/actions';
+import { fetch, onSuccess, onError, fromApi } from '../';
+
+// we are mocking impure-lodash here, so that conciergeInitialFetchError() will contain the expected id in the tests
+jest.mock( 'lib/impure-lodash', () => ( {
+	uniqueId: () => 'mock-unique-id',
+} ) );
```

Comment:
Do we really need that mock? I'm not sure to understand how this test relates to `conciergeInitialFetchError()`.

Reply:
Ah! Good catch. It's not necessary and the comment was a copy-and-paste error. Fixed in f086733.